data/helpers.d/system

```diff
@@ -53,3 +53,108 @@ ynh_abort_if_errors () {
 ynh_get_debian_release () {
 	echo $(lsb_release --codename --short)
 }
+
+# Read the value of a key in a ynh manifest file
+#
+# usage: ynh_read_manifest manifest key
+# | arg: -m, --manifest= - Path of the manifest to read
+# | arg: -k, --key= - Name of the key to find
+ynh_read_manifest () {
+	# Declare an array to define the options of this helper.
+        declare -Ar args_array=( [m]=manifest= [k]=manifest_key= )
+        local manifest
+        local manifest_key
+	# Manage arguments with getopts
+	ynh_handle_getopts_args "$@"
+
+	python3 -c "import sys, json;print(json.load(open('$manifest', encoding='utf-8'))['$manifest_key'])"
```

Comment:
We should probably just bundle [jq](https://stedolan.github.io/jq/) with YunoHost, that would make things easier (quite minor difference I know).

It's packaged into debian.

Reply:
Other comment: do we have a guarantee that python THREE is installed? We don't use it in YunoHost I think :thinking:

Reply:
I've no idea...
This helper isn't from me. I'd rather prefer a good old "grep | cut" which is faster than that.
But, this helper is already used by some apps, and it seems to work fine so far.

Reply:
Mybe jq could also fix definitely any problem about encoding. I rememberer I had some problem about encoding in synapse (this problem is solved by `encoding='utf-8'`) and it's why I decided to use python3 which has a better support of encoding.

Reply:
jq is now a dependacy of yunohost if you wish to uses it https://github.com/YunoHost/yunohost/pull/532